astro/src/content/quickstarts/quickstart-springboot-web.mdx

```diff
@@ -56,188 +41,258 @@ The login flow will look like this after FusionAuth is introduced.
 
-The properties with the `spring.security.oauth2.client.registration` prefix configure the settings for the OAuth2 Client. The `client-id`, `client-secret`, `redirect-uri` and `authorization-grant-type` all need to match what is configured in FusionAuth. If you used the Kickstart method these settings should be correct, however if they are not you can log in to http://localhost:9011/admin and check the [OAuth Settings](/docs/v1/tech/core-concepts/applications#oauth) for your application.
+The properties with the `spring.security.oauth2.client.registration` prefix configure the settings for the OAuth2 Client. The `client-id`, `client-secret`, `redirect-uri`, and `authorization-grant-type` values all need to match what is configured in FusionAuth. Because you used the Kickstart method, these settings should be correct, but if they are not, you can log in to [FusionAuth admin UI](http://localhost:9011/admin) and check the [OAuth Settings](/docs/v1/tech/core-concepts/applications#oauth) for your application.
 
 The properties with the `spring.security.oauth2.provider` prefix tell Spring where the important endpoints for FusionAuth are.
 
-### Add controllers
+### Add Controllers
 
-Find the `base directory` when the main application java class is defined. In our example this is `src/main/java/io/quickstart/springweb` and the main class is `FusionAuthSpringApplication.java`.
+Find the base directory where the main application Java class is defined. In our example, the main class is `FusionAuthSpringApplication.java` in `src/main/java/io/quickstart/springweb`.
 
-Create the following new java classes in the `base directory`:
+<Aside type="note">If you downloaded your own starter, you will need to update the `package` for the code snippets below to match your own.</Aside>
+
+<RemoteCode url="https://raw.githubusercontent.com/FusionAuth/fusionauth-quickstart-springboot-web/main/complete-application/src/main/java/io/fusionauth/quickstart/springweb/FusionAuthSpringApplication.java"
+            lang="java"/>
+
+Create the following `HomeController.java` Java class in the `src/main/java/io/quickstart/springweb` directory.
 
-`HomeController.java`
 
 <RemoteCode url="https://raw.githubusercontent.com/FusionAuth/fusionauth-quickstart-springboot-web/main/complete-application/src/main/java/io/fusionauth/quickstart/springweb/HomeController.java"
             lang="java"/>
 
-This controller serves the page for unauthenticated users on the `/` route and shows the application name which is injected from the `spring.application.name` property in the `application.properties` file. The return string `home` tells Spring to render the `home.html` template.
+This controller serves the page for unauthenticated users on the `/` route and shows the application name, which is injected from the `spring.application.name` property in the `application.properties` file. The `home` return string tells Spring to render the `home.html` template.
```

Comment:
where is the home.html file located?  I see that this created later, so you will want to say something like "...`home.htm` template which will be created later."